webui/src/pages/auth/groups/index.tsx

```diff
@@ -146,6 +158,18 @@ const GroupsContainer = () => {
                     </ConfirmationButton>
                 </ActionGroup>
                 <ActionGroup orientation="right">
+                    <InputGroup>
```

Comment:
Have you considered creating a component from this?
Something like `<SearchInput>`, that can be used for policies as well, and also other components in the future?